### PR TITLE
Convert pipelinewise repo fork to look more DEG-like

### PR DESCRIPTION
**Pull request by pnadolny13**
- add tox
- stop using nose
- use our circleci workflow
- fix styling that doesnt pass our codestyles test
- populate the README so its more clear how to use the thing in a standard singer fashion rather than the pipelinewise way

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+[testenv]
+setenv =
+    ENV = dev
+passenv =
+    ENV
+    TAP_SNOWFLAKE_USER
+    TAP_SNOWFLAKE_ACCOUNT
+    TAP_SNOWFLAKE_PASSWORD
+    TAP_SNOWFLAKE_WAREHOUSE
+    TAP_SNOWFLAKE_DBNAME
```
Comment (cwatkins989) [inline tox.ini:13]: Does a test DB actually exist somewhere? If so, maybe we should update the README on where that DB is and how it is being maintained.

File: README.md [hunk 2]
```diff
@@ -106,18 +142,20 @@ specified in the table's metadata as well.
   export TAP_SNOWFLAKE_WAREHOUSE=<snowflake-warehouse>
 ```
 
-2. Install python dependencies in a virtual env and run nose unit and integration tests
+NOTE: User must have permission to create/drop schema (TAP_SNOWFLAKE_TEST), create/drop table, insert into those tables
```
Comment (cwatkins989) [inline README.md:145]: Referring to my earlier note - saw this was updated. Maybe just make it clear which DB we are using in Snowflake for this.

**Review by cwatkins989 — APPROVED** [review]
Looks great - found the underlying Snowflake DB being used for this. Maybe just a bit more specificity in README? But certainly not blocking.

File: README.md (hunk 2 above)
Comment (pnadolny13) [reply README.md:145]: Its DATA_DEG_TEST.TAP_SNOWFLAKE_TEST. I can add it to the readme

**Pull request merged**